src/config/ReportConfig.tsx

```diff
@@ -636,6 +636,12 @@ export const REPORT_TYPES = {
     },
     maxRecords: 250,
     settings: {
+      type: {
```

Comment:
Can you add this to the docs too? 
[
docs/modules](https://github.com/neo4j-labs/neodash/blob/master/docs/modules/ROOT/pages/user-guide/reports/line-chart.adoc)